src/graphql/schema.ts

```diff
@@ -1,9 +1,19 @@
 import { makeExecutableSchema } from 'graphql-tools';
+import { nodeInterface, pageInfoType } from 'graphql-relay-tools';
 import { importSchema } from 'graphql-import';
 
 import resolvers from './resolvers';
 
-const typeDefs: string = importSchema('src/graphql/schema/schema.graphql');
+/** SWAPI schema */
+import { swapiDef } from './schema/swapi';
```

Comment:
please covert all `ts` schema files to `graphql` SDL (sorry)

Reply:
It's not possible due to Relay connections. Please see ts schema files for details.